spec/features/team/invite_a_team_member_spec.rb

```diff
@@ -64,14 +66,40 @@
       end
 
       context "with an email that already exists" do
-        let(:invited_user_email) { user.email }
+        context "and is confirmed" do
+          let(:invited_user_email) { user.email }
+
+          it "tells the user that the email has already been taken" do
+            expect {
+              click_on "Send invitation email"
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(0)
+            expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+          end
+        end
 
-        it "tells the user that the email has already been taken" do
-          expect {
-            click_on "Send invitation email"
-          }.to change { User.count }.by(0)
-          expect(InviteUseCaseSpy.invite_count).to eq(0)
-          expect(page).to have_content("Email is already associated with an account. If you can't sign in, reset your password")
+        context "and is not confirmed" do
+          before do
+            sign_out
+            sign_up_for_account(email: invited_user_email)
+            sign_in_user user
+            visit root_path
+            click_on "Team"
+            click_on "Invite team member"
+            fill_in "Email", with: invited_user_email
+          end
+
+          let(:invited_user_email) { 'notconfirmedyet@gov.uk' }
+          let(:invited_user) { User.find_by(email: invited_user_email) }
+
+          it "sends them an invite to the organisation's account" do
+            expect {
+              click_on "Send invitation email"
+            }.to change { User.count }.by(0)
+            expect(InviteUseCaseSpy.invite_count).to eq(1)
+            expect(invited_user.confirmed?).to eq(false)
+            expect(invited_user.organisation_id).to eq(user.organisation_id)
+          end
```

Comment:
Still missing the example when the user already has an organisation. We don't want to be deleting users that have already been invited to an organisation.